RFC-007-SWAP-Ethereum-basic.md

```diff
@@ -0,0 +1,315 @@
+# Ethereum Basic HTLC Atomic Swap
+
+- RFC-Number: 007
+- Status: Draft
+- Discussion-issue: -
+- Created on: 2019-03-12
+
+## Table of Contents
+<!-- toc -->
+- [Description](#description)
+- [The Ethereum Identity](#the-ethereum-identity)
+- [Hash Time Lock Contract](#hash-time-lock-contract)
+    - [Hash Functions](#hash-functions)
+    - [Parameters](#parameters)
+    - [Contract](#contract)
+- [Execution Phase](#execution-phase)
+    - [Deployment](#deployment)
+    - [Redeem](#redeem)
+    - [Refund](#refund)
+- [Registry extension](#registry-extension)
+- [Examples](#examples)
+    - [RFC003 SWAP REQUEST](#rfc003-swap-request)
+    - [RFC003 SWAP RESPONSE](#rfc003-swap-response)
+    - [HTLC](#htlc)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC defines how to execute a [RFC003](./RFC-003-SWAP-basic.md) SWAP where one of the ledgers is Ethereum and the associated asset is the native Ether asset.
+
+For definitions of the Ethereum ledger and Ether asset see [RFC006](./RFC-004-SWAP-Ethereum.md).
+
+To fulfil the requirements of [RFC003](./RFC-003-SWAP-basic.md) this RFC defines:
+
+- The [identity](./RFC-003-SWAP-basic.md#identity) to be used when negotiating a SWAP on the Ethereum ledger.
+- How to construct a Hash Time Lock Contract (HTLC) to lock the Ether asset on the Ethereum blockchain.
```

Comment:
```suggestion
- How to construct a Hashed TimeLock Contract (HTLC) to lock the Ether asset on the Ethereum blockchain.
```

This is how it is written in previous RFCs.

Reply:
It isn't though🤨

Reply:
I guess it *was*, I did a search locally before saying that....